aries-site/src/pages/components/layer.mdx

```diff
@@ -106,3 +107,33 @@ A center modal is anchored at the center of the window and acts as a modal. This
 >
   <LayerCenterExample />
 </Example>
+
+### Layer Form with sticky Header
+
+The Layer header and action buttons can be sticky while the rest of the Layer content is scrollable. This can be accomblished by wrapping a Form within a Box with `overflow` set to `auto`.
+
+<Example
+  docs="https://v2.grommet.io/layer?theme=hpe#props"
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/LayerStickyExample.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/MonitorFormExample.js"
+  ]}
+  height={{ min: 'small' }}
+>
+  <LayerStickyExample />
+</Example>
+
+### Layer Form with hide/show sticky Header
+
+In order to maximize screen real-estate when content is scrollable, the header may scroll away as the user scrolls down but reappear as the user scrolls up.
```

Comment:
Should we be mentioning using Header with `sticky="scrollup"` ?

Reply:
done